data/yam/agama/auto/sles_default_keyboard_layout.jsonnet

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
no need for "default" it isn't, please drop it from the name.

Reply:
a better name would be: `sles_change_localization.jsonnet` you can apply that name to all the files in this PR. otherwise is confusing, because we are not only changing the keyboard but the language too.

Reply:
Done, thanks for review.